Getting-Started/Setup/Server-Setup/Load-Balancing/flexible.md

```diff
@@ -38,9 +38,31 @@ The process is as follows:
 * When a front-end server detects that there are pending instructions, it downloads them and processes them and in turn updates it's cache, cache files and indexes on it's own file system
 * There can be up to a 5 second delay between content updates and a front-end server's refreshing, this is expected and normal behavior.
 
-## Configuration files
+## Umbraco Configuration files
 
-There isn't any Umbraco configuration file changes necessary! **You must not enable the distributed calls flag in the umbracoSettings.config** file for Flexible Load Balancing to work, that is purely for Traditional load balancing. There are some Examine/Logging config file updates needed (see below and the [Overview](index.md))
+There isn't any _Umbraco_ configuration file changes necessary. **You must not enable the distributed calls flag in the umbracoSettings.config** file for Flexible Load Balancing to work, that is purely for Traditional load balancing. 
+
+There are some Examine/Logging config file updates needed (see below and the [Overview](index.md))
+
+##Internal Server Requests
+
+It is required that an Umbraco website instances can make internal requests to itself.
+This is required for 3 things:
+
+* Keep alive service - to ensure scheduled publishing occurs
+* Scheduled tasks - to initiate any configured scheduled tasks
+* Scheduled publishing - to initiate any scheduled publishing for documents
+
+Flexible Load Balancing will automatically elect a 'master' scheduling server to perform the above services. This means
+that all of the servers will need to be able to resolve either: itself or the load balanced front-end url.
+
+By default, Umbraco will globally set the 'base url' of the website to the address made by the first request when the AppDomain starts and it is assumed
+that your website can resolve that DNS address to itself or the main load balancer. 
+In many scenarios this is fine, but in case this is not adequate there's a few of options you can use:
+
+* set the `umbracoApplicationUrl` property in the [Web.Routing section of /Config/umbracoSettings.config](https://our.umbraco.org/Documentation/Reference/Config/umbracoSettings/)
+* or in an [`ApplicationStarting` event of an application startup handler](https://our.umbraco.org/Documentation/Reference/Events/Application-Startup) you can specify a custom delegate to return the base url for a node by setting [`ApplicationUrlHelper.ApplicationUrlProvider`](https://github.com/umbraco/Umbraco-CMS/blob/75c2b07ad3a093b5b65b6ebd45697687c062f62a/src/Umbraco.Core/Sync/ApplicationUrlHelper.cs#L21)
+* [can set your front-end(s) (non-admin server) to be explicit slave servers](https://our.umbraco.org/documentation/Getting-Started/Setup/Server-Setup/load-balancing/flexible-advanced) which means they will never be used as the master scheduler 
```

Comment:
done, the online editor is by far the best tool for getting the relative links right :)